src/helpers/image-upload.js

```diff
@@ -0,0 +1,18 @@
+import cloudinary from '../config/cloudinary-config';
+
+export class ImageUpload {
+  static uploadImages = async (filePaths) => {
+    const results = [];
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < filePaths.length; i++) {
+      // eslint-disable-next-line no-await-in-loop
+      await cloudinary.uploader.upload(filePaths[i], (error, result) => {
+        if (result) {
+          results.push(result);
+        }
+        console.log(error);
```

Comment:
do we need this console statement

Reply:
No, we don't. Thanks for the observation